internal/app/backend/backend_test.go

```diff
@@ -17,17 +17,14 @@ package backend
 import (
 	"testing"
 
-	"google.golang.org/grpc"
-	"open-match.dev/open-match/internal/pb"
+	"open-match.dev/open-match/internal/config"
 	"open-match.dev/open-match/internal/rpc"
 	rpcTesting "open-match.dev/open-match/internal/rpc/testing"
 )
 
 func TestServerBinding(t *testing.T) {
-	bs := func(p *rpc.ServerParams) {
-		p.AddHandleFunc(func(s *grpc.Server) {
-			pb.RegisterBackendServer(s, &backendService{})
-		}, pb.RegisterBackendHandlerFromEndpoint)
+	bs := func(p *rpc.ServerParams, cfg config.Mutable) {
+		BindService(p, cfg)
 	}
 
 	rpcTesting.TestServerBinding(t, bs)
```

Comment:
It seems you should be able to do rpcTesting.TestServerBinding(t, BindService)
Though BindService returns err, so it won't fit.  But that err isn't being testing when it should be?

Reply:
Ah, I see, the config here is mutable, instead of just a view.
It seems that everything should be done in two steps, not one:
- Do all configuration
- Then do all binding.

Otherwise will result in possible bugs from ordering problems.  I see that miniredis doesn't like this as is, but looking at the documentation, it is possible to tell mini redis the host and port to serve on: https://godoc.org/github.com/alicebob/miniredis#Miniredis.StartAddr